pages/api/search/issues/index.ts

```diff
@@ -34,28 +44,41 @@ async function get(req: NextApiRequest, res: NextApiResponse) {
   }
   const include = [
     { association: 'developers' },
-    { association: 'pullRequests' },
+    {
+      association: 'pullRequests',
+      ...(pullRequester ? { where: { githubLogin: pullRequester } } : {})
+    },
     { association: 'mergeProposals' },
     { association: 'repository' }
   ]
 
-  let issues = await models.issue.findAll({
+  const issues = await models.issue.findAll({
     where: whereCondition,
     include,
     nest: true,
     order: [[req.query.sortBy || 'createdAt', req.query.order || 'DESC']]
   })
 
+  const result = []
+
   if (search)
-    issues = issues.filter(
-      (issue) =>
-        searchPatternInText(issue.title, String(search)) ||
-        searchPatternInText(issue.body, String(search))
+    result.push(
+      ...issues.filter(
+        (issue) =>
```

Comment:
can be abreviated to `...issues.filter( ({title, body}) => [title, body].some( text => searchPatterInText(text) ) )`